livre/core/dash/DashRenderNode.h

```diff
@@ -70,7 +70,7 @@ class DashRenderNode
      * @return True, if object is visible.
      * @warning This condition is set from outside of the object.
      */
-    LIVRECORE_API bool isLODVisible() const;
```

Comment:
I clearly remember this function name has been changed on purpose. It was isVisible() before but the functionality is really the LOD is visible.